packages/slice-machine/lib/builders/common/Zone/Card/components/Hints/Renderers/react.tsx

```diff
@@ -95,11 +95,7 @@ const codeByWidgetType = (
       useKey ? appendKey("color") : ""
     } style={{ color: ${fieldText} }}>Some Text</span>`,
 
-  [Widgets.Text?.TYPE_NAME]: (
-    fieldText: string,
-    useKey?: boolean
-  ) => `/* import { PrismicText } from '@prismicio/react' */
-<PrismicText ${useKey ? appendKey("text") : ""} field={${fieldText}} />`,
+  [Widgets.Text?.TYPE_NAME]: (fieldText: string) => `{${fieldText}}`,
```

Comment:
`ceateDefaultField` would work here too?

Reply:
<img width="371" alt="Screenshot 2022-06-21 at 14 58 39" src="https://user-images.githubusercontent.com/69536010/174817859-b13d3697-ed60-444c-a549-27175c7c1a53.png">
Create default would gives us this ⬆️ . According to the slack messages we want this? ⬇️
<img width="483" alt="Screenshot 2022-06-21 at 14 59 25" src="https://user-images.githubusercontent.com/69536010/174818031-be084b90-b197-4ddd-b2cc-01331a29423f.png">

Reply:
Sure will change! :)